client/state/notices/actions.js

```diff
@@ -38,6 +41,13 @@ function createNotice( status, text, options = {} ) {
 	}
 }
 
+export function setRoute( path ) {
```

Comment:
It is, for the notice to disappear when navigating to new route
https://github.com/Automattic/wp-calypso/pull/2009/files#diff-77b3f6be0290b9520a2b10b906721494R23

It is triggered in `boot/index`:
https://github.com/Automattic/wp-calypso/pull/2009/files#diff-77b3f6be0290b9520a2b10b906721494R23